gateway/server.go

```diff
@@ -421,7 +424,9 @@ func (gw *Gateway) setupGlobals() {
 		NewRelicApplication = gw.SetupNewRelic()
 	}
 
-	gw.readGraphqlPlaygroundTemplate()
+	if playgroundTemplate == nil {
```

Comment:
I'd add an additional condition of spec.GraphQL.GraphQLPlayground.Enabled, we already know it should be behind that, or move the block to graphql scope, use a sync.Once...